receiver/kymastatsreceiver/internal/modulediscovery/client.go

```diff
@@ -10,20 +10,31 @@ import (
 	"k8s.io/client-go/discovery"
 )
 
+type Config struct {
+	//ModuleGroups are API groups of Kyma modules to be discovered.
+	ModuleGroups []string `mapstructure:"module_groups"`
+
+	//ExludedResources are API resource names to be excluded when discovering Kyma modules.
+	ExludedResources []string `mapstructure:"excluded_resources"`
```

Comment:
There is a type here: `ExcludedResources` instead of `ExludedResources`

Reply:
Good catch! Will fix

Reply:
Fixed